rotkehlchen/chain/evm/decoding/omnibridge/decoder.py

```diff
@@ -0,0 +1,135 @@
+            to_address = from_address  # We have no to_address information
+        elif context.tx_log.topics[0] == self.withdrawal_topic:
+            to_address = hex_or_bytes_to_address(context.tx_log.topics[2])
+            from_address = to_address  # We have no from_address information
+        else:
+            return DEFAULT_DECODING_OUTPUT
+
+        bridged_asset = get_or_create_evm_token(
+            userdb=self.evm_inquirer.database,
+            evm_address=hex_or_bytes_to_address(context.tx_log.topics[1]),
+            chain_id=self.evm_inquirer.chain_id,
+            evm_inquirer=self.evm_inquirer,
+        )
+        amount = asset_normalized_value(
+            amount=hex_or_bytes_to_int(context.tx_log.data[0:32]),
+            asset=bridged_asset,
+        )
+        expected_event_type, new_event_type, from_chain, to_chain, expected_location_label = bridge_prepare_data(  # noqa: E501
+            tx_log=context.tx_log,
+            deposit_topics=(self.deposit_topic,),
+            source_chain=self.source_chain,
+            target_chain=self.target_chain,
+            from_address=from_address,
+            to_address=to_address,
+        )
+
+        for event in context.decoded_events:
+            if bridged_asset == A_WETH and event.asset == A_ETH:
+                if event.event_type == HistoryEventType.RECEIVE:
+                    expected_location_label = context.transaction.from_address
+                    from_address = expected_location_label
+                    to_address = expected_location_label
+                expected_address = hex_or_bytes_to_address(context.tx_log.topics[2])
+                expected_asset = A_ETH
+            else:
+                expected_address = self.bridge_address
+                expected_asset = bridged_asset
+
+            if (
+                event.event_type == expected_event_type and
+                event.location_label == expected_location_label and
+                event.address in (expected_address, ZERO_ADDRESS) and
+                event.asset == expected_asset and
+                (amount is None or event.balance.amount == amount)
+            ):
+                bridge_match_transfer(
+                    event=event,
+                    from_address=from_address,
+                    to_address=to_address,
+                    from_chain=from_chain,
+                    to_chain=to_chain,
+                    amount=event.balance.amount,
+                    asset=bridged_asset,
+                    expected_event_type=expected_event_type,
+                    new_event_type=new_event_type,
+                    counterparty=GNOSIS_CPT_DETAILS,
+                )
+                break
+        else:
+            log.error(
+                f'Could not find the transfer event for bridging to {to_address}'
+                f' in {self.evm_inquirer.chain_name} transaction {context.transaction.tx_hash.hex()}',  # noqa: E501
```

Comment:
double check it but I believe that the transaction class overwrites the `__str__` method to display the hash and the chain

```suggestion
                f' in {context.transaction}',  # noqa: E501
```